tests/test_mavis/summary/test_summary.py

```diff
@@ -174,3 +216,32 @@ def test_filtering_events_split(self):
     @todo
     def test_get_pairing_state(self):
         pass
+
+
+class TestFilterByCallMethod:
+    def test_annotate_dgv_multiple_match(self, genomic_event3, dgv_event):
+        bpps = [genomic_event3]
+        annotate_dgv(bpps, dgv_event, 100)
+        assert bpps[0].data["known_sv_count"] == 2
+        output_dgv_tracking_id = list(bpps[0].data["dgv"].split(','))
+        expected = ['dgv1n82', 'rgv2n98']
+        for output_tracking_id in output_dgv_tracking_id:
+            assert output_tracking_id in expected
+        assert len(bpps) == 1
+
+    def test_annotate_dgv_no_match(self, genomic_event4, dgv_event):
+        bpps = [genomic_event4]
+        annotate_dgv(bpps, dgv_event, 100)
+        assert bpps[0].data["known_sv_count"] == 0
+        assert bpps[0].data["dgv"] == ''
+        assert len(bpps) == 1
```

Comment:
It doesn't makes sense to test the length of bpps since it was defined above and the function does not change the input list

Reply:
Agreed, removed in [473afbc](https://github.com/bcgsc/mavis/pull/336/commits/473afbc04ece54e5d4fdb5ebdea6b4b7df4d9d40)